api/namex/utils/common.py

```diff
@@ -96,6 +99,24 @@ def convert_to_ascii(value):
         return value
 
 
+def convert_to_utc_min_date_time(date_str: str):
+    date_obj = datetime.strptime(date_str, '%Y-%m-%d').date()
+    min_time = time(hour=0, minute=0, second=0, microsecond=0)
+    return datetime.combine(date_obj, min_time, tzinfo=tz.UTC)
```

Comment:
This changes the date string to, say "2021-04-17 00:00:00 Pacific" right? (ie, not 00:00 UTC)

Reply:
This should be converting to UTC properly now with the latest updates.

![image](https://user-images.githubusercontent.com/6685223/118575394-37ad7480-b73b-11eb-9f17-4cdfdbf9a1f1.png)